uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup/leave-request-popup-details-tab.html

```diff
@@ -84,7 +84,7 @@
 </crm-loading>
 
 <!--Expiry Date Section - Only For Toil-->
-<div class="row" ng-show="$ctrl.isLeaveType('toil')">
+<div class="row" ng-show="$ctrl.isLeaveType('toil') && $ctrl.requestCanExpire">
```

Comment:
You have changed the `initRequestCanExpire` function so it returns the opposite value, but the check of `$ctrl.requestCanExpire` in the template still remains the same. Was here a bug in the previous commit?